src/modules/timer.rs

```diff
@@ -0,0 +1,83 @@
+use crate::config::TableExt;
+use ansi_term::Color;
+
+use super::{Context, Module};
+
+/// Outputs the time it took the last command to execute
+///
+/// Will only print if last command took more than two seconds to execute.
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("timer")?;
+    let arguments = &context.arguments;
+    let elapsed = arguments
+        .value_of("elapsed_time")
+        .unwrap_or("x") // not a time
+        .parse::<u64>()
+        .ok()?;
+
+    // Question: Is this a good series of calls to use? Is there a cleaner way to set types?
+    // Question: If the input config is negative, should I warn the user?
+    let config_min = context
+        .config
+        .get_module_config("timer")
+        .and_then(|table| table.get_config("min_time"))
+        .and_then(|value| value.as_integer())
```

Comment:
Ideally, I think we should have a `get_as_int` accessor with the other config accessors:
https://github.com/starship/starship/blob/af1b66708abc462fe0db34083c3f6c1d790f61a3/src/config.rs#L12-L14 (this is on the branch where `TableExt` is unified with `Config`)

Reply:
Implemented. I've called it get_as_i64 since it wasn't clear to me initially what type of int would be returned.